### PR TITLE
Month 1 - Bugfixes

### PR DESCRIPTION
**Pull request by seomasha**

### CODE REVIEW

File: cinemaapp_frontend/src/pages/MovieDetails.jsx [hunk 1]
```diff
@@ -73,6 +71,8 @@ const MovieDetails = () => {
   const dayPickers = [];
   for (let i = 0; i < 10; i++) {
     nextDate.setDate(today.getDate() + i);
+    const date = nextDate.toLocaleDateString("en-US", dateOptions);
+    const day = nextDate.toLocaleDateString("en-US", dayOptions);
```
Comment (ivanazirojevic-abh) [inline cinemaapp_frontend/src/pages/MovieDetails.jsx:75]: why do you need this in the loop?

Comment (seomasha) [reply cinemaapp_frontend/src/pages/MovieDetails.jsx:75]: If that is out of the loop, that means that basically the 10 day pickers will show just the todays date.

File: cinemaapp_backend/src/main/java/com/abhinternship/CinemaApp/service/VenueServiceImpl.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public VenueDTO findAllVenues(final int page, final int size) {
 
             return new VenueDTO(venues.getContent(), totalSize);
         }
-    }
+    } // da li se metoda pozvala i koja
```
Comment (ivanazirojevic-abh) [inline cinemaapp_backend/src/main/java/com/abhinternship/CinemaApp/service/VenueServiceImpl.java]: remove comment

File: README.md [hunk 3]
```diff
@@ -106,7 +114,7 @@ To fetch mock movie data while developing the application, you can set up a JSON
    npm install -g json-server
 2. Run the following command to start the JSON server on port **8888**:
    ```bash
-   json-server --watch data/mock-movie.json --port 888
+   json-server --watch data/mock-movie.json --port 8888
```
Comment (ivanazirojevic-abh) [inline README.md]: is json-server still in use?

File: cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    @Test
+    public void testFindCurrentlyShowingMovies_NoFilter() {
+        final Page<Movie> moviePage = new PageImpl<>(List.of(testMovie));
+
+        when(movieRepository.findByProjectionStartDateBeforeAndProjectionEndDateAfter(any(LocalDate.class), any(LocalDate.class), eq(pageable)))
+                .thenReturn(moviePage);
+        when(projectionRepository.findByMovieIdOrderByProjectionTime(any(Movie.class)))
+                .thenReturn(Set.of(testProjection));
+
+        final FilterMovie filterMovie = FilterMovie.empty();
```
Comment (ivanazirojevic-abh) [inline cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java:78]: Remind me what empty() does, please

File: cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        final MovieListDTO result = movieService.findCurrentlyShowingMovies(filterMovie, 0, 10);
+
+        assertEquals(1, result.getMovies().size());
+        assertEquals("Test Movie", result.getMovies().getFirst().getName());
```
Comment (ivanazirojevic-abh) [inline cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java]: Make "Test Movie" a variable and use it in all tests.

File: cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/utils/FilterMovieTest.java [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        filters = new HashMap<>();
+        parameters = new HashMap<>();
+    }
+
+    @Test
+    void testToQueryStringWithVenueId() {
+        filters.put("venueId", "1");
+        FilterMovie filterMovie = new FilterMovie(filters);
+
+        String queryString = filterMovie.toQueryString(parameters, true);
```
Comment (ivanazirojevic-abh) [inline cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/utils/FilterMovieTest.java]: Make all variables in this file final

File: cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java (hunk 4 above)
Comment (seomasha) [reply cinemaapp_backend/src/test/java/com/abhinternship/CinemaApp/service/MovieServiceTest.java:78]: It just returns an empty map.

File: README.md (hunk 3 above)
Comment (seomasha) [reply README.md]: Nope, it is not.

Comment (ivanazirojevic-abh) [reply README.md]: Remove it from readme and from dependencies

**Review by ivanazirojevic-abh — APPROVED** [review]
All looks great! Let's merge this PR for now and we can open another one for bugs when we find them.

**Pull request merged**